src/main/java/com/terraformersmc/terrestria/init/TerrestriaFeatures.java

```diff
@@ -172,6 +187,7 @@ public static void init() {
 				new VolcanoStructureFeature(DefaultFeatureConfig::deserialize)
 		);
 
+		//TODO test and change this to volcano to be standard case with other features
```

Comment:
space between // and text

Reply:
No it didn't because one is for BareSmallLog trees where the other is for ones with leaves